addons/website/static/src/components/fields/fields.js

```diff
@@ -16,15 +16,21 @@ const {Component, useState, onWillStart} = owl;
  */
 class PageUrlField extends Component {
     setup() {
+        this.orm = useService('orm');
+        this.serverUrl = `${window.location.origin}/`;
+        this.pageUrl = this.fieldURL;
+        this.pageMimetypes = {};
+
         this.state = useState({
             redirect_old_url: false,
-            url: this.props.value,
+            url: this.pageUrl,
             redirect_type: '301',
         });
-        useInputField({getValue: () => this.props.value.url || this.props.value});
 
-        this.serverUrl = window.location.origin;
-        this.pageUrl = this.props.value;
+        onWillStart(async () => {
+            this.pageMimetypes = await this.orm.call('website', 'guess_mimetype', []);
+        });
```

Comment:
What is this `pageMimetypes` used for?

Reply:
Thanks for spotting this! this is a dead code now since I removed JS slugify... removed :+1: